modules/core/src/test/java/org/apache/ignite/internal/processors/cache/CacheStopAndDestroySelfTest.java

```diff
@@ -181,20 +177,6 @@ private CacheConfiguration getNearConfig() {
         return cfg;
     }
 
-    /**
```

Comment:
upper code :
    /** Local cache name. */
    private static String CACHE_NAME_LOC = "cache_local";
Seems not used, remove it plz

Reply:
Fixed.